src/app/modular/post/reducer.ts

```diff
@@ -20,6 +21,15 @@ const reducer: PostReducer = produce(
       case actions.LOADING_START:
         state.loading = true;
         break;
+      case actions.SAVE_POSTS:
+        state.posts = action.payload.posts;
+        break;
+      case actions.LOADING_SUCCESS:
+        state.loading = false;
+        break;
+      case actions.LOADING_ERROR:
```

Comment:
These can be combined into a single statement 
```suggestion
      case actions.LOADING_SUCCESS:
      case actions.LOADING_ERROR:
```